examples/helm/ceph/templates/exporter/deployment.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.ceph.enabled.exporter }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: ceph-exporter
+  labels:
+    version: {{ .Chart.Version }}
+    app: ceph
+    daemon: exporter
+    release: {{ .Release.Name }}
+  annotations:
+    prometheus.io/scrape: 'true'
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        version: {{ .Chart.Version }}
+        app: ceph
+        daemon: exporter
+        release: {{ .Release.Name }}
+    spec:
+      nodeSelector:
+        {{ .Values.labels.node_selector_key }}: {{ .Values.labels.node_selector_value | quote }}
+      containers:
+      - name: ceph-exporter
+        image: {{ .Values.images.exporter }}
+        imagePullPolicy: {{ .Values.image_policy.pull }}
+        ports:
+          - name: ceph-exporter
+            containerPort: {{ .Values.service.exporter.port }}
+        volumeMounts:
+          - name: ceph-conf
+            mountPath: /etc/ceph/ceph.conf
```

Comment:
Possible but may need some rethinking across all of the resources in the helm chart. Something that can be done in another PR?